api/cpp/docs/cmake_reference.md

```diff
@@ -88,3 +88,14 @@ set_property(TARGET my_application PROPERTY SLINT_BUNDLE_TRANSLATIONS "${CMAKE_C
 By default, the domain used for translations is the name of the CMake target the `.slint` files are targeted with.
 Use the `SLINT_TRANSLATION_DOMAIN` target property to override this and use the specified value as domain, instead.
 This is useful in build environments where the target name is given and not suitable, such as esp-idf.
+
+## Live-Preview
+
+By default, `.slint` files are compiled to C++ code when using the `slint_target_sources()` function. This is recommended
+for release builds.
+
+During the debugging and development phase, you can opt into enabling live-preview for your running application:
+
+Enable the `SLINT_FEATURE_LIVE_PREVIEW` cmake option when compiling Slint and set the `SLINT_LIVE_PREVIEW=1` environment
```

Comment:
Note that this should be enabled when compiling Slint, not the application.
this might be confusing if one use the binaries.